### ISSUE TITLE
VictimCrasherCompact Overlay text are misaligned when values are higher than 10

### ISSUE THREAD

**Issue by Afireee** [Piotrekol/StreamCompanion-overlays#1]
this image should explain everything https://i.imgur.com/bxyegdo.png

this also what it looks like when playing in-game: https://i.imgur.com/FRIlhFY.png

**Comment by Piotrekol** [comment]
> this also what it looks like when playing in-game: i.imgur.com/FRIlhFY.png

What mods do you have enabled when playing? just HDDT?

> this image should explain everything i.imgur.com/bxyegdo.png

I'll have to check this myself later today

**Comment by Afireee** [comment]
> What mods do you have enabled when playing? just HDDT?

Generally, +DT can make the values go higher than 10. I don't think it applies to +HR only, BUT +HRDT also applies (https://i.imgur.com/c09nH6t.png)

**Comment by Piotrekol** [comment]
Yes, I'm aware of that :). I was asking because of this image with invalid URL being displayed 
![image](https://user-images.githubusercontent.com/4990365/148970250-f8e8e767-8096-43bd-8db9-6012b5afba21.png)

**Comment by Afireee** [comment]
> Yes, I'm aware of that :). I was asking because of this image with invalid URL being displayed ![image](https://user-images.githubusercontent.com/4990365/148970250-f8e8e767-8096-43bd-8db9-6012b5afba21.png)

Ah, that was with +Auto mod, it didn't show because the keyword written by default in .js file is "at". I fixed it by changing it to "au"
![image](https://user-images.githubusercontent.com/28188083/148971436-91b580f1-ea31-4a3c-bd38-7627a7c8e1b9.png)
